src/Chainweb/RestAPI/Orphans.hs

```diff
@@ -431,17 +432,46 @@ deriving instance ToSchema a => ToSchema (Time a)
 deriving instance ToSchema a => ToSchema (TimeSpan a)
 
 instance ToSchema ChainwebVersion where
-  declareNamedSchema _ = pure $ NamedSchema (Just "ChainwebVersion") mempty
+    declareNamedSchema _ = pure . NamedSchema (Just "ChainwebVersion") $ mempty
+        & description ?~ "Unique identifier for a Chainweb network"
+        & type_ .~ SwaggerString
+        & example ?~ toJSON Testnet02
 
 instance ToSchema MerkleLogHash where
-  declareNamedSchema _ = pure $ NamedSchema (Just "MerkleLogHash") mempty
+    declareNamedSchema _ = pure $ NamedSchema (Just "MerkleLogHash") mempty
 
--- TODO Need more detail for these two!
 instance ToSchema HeaderBytes where
-  declareNamedSchema _ = pure $ NamedSchema (Just "HeaderBytes") mempty
+    declareNamedSchema _ = pure . NamedSchema (Just "HeaderBytes") $ binarySchema
+        & description ?~ "An encoded BlockHeader"
+        & minLength ?~ 302
+        & maxLength ?~ 302
```

Comment:
This number depends on the chain web version. While the chainweb version is part of the API type, it's not available here.

Reply:
I got `302` from the Wiki. Is it still a safe default for now?

Reply:
It's valid for Peterson graph and twenty chain (both 3 regular) with 32 bit hashes.

Reply:
So, currently it holds for devnet and testnet